tools/bliss-to-ogg-zip.py

```diff
@@ -9,10 +9,12 @@
 from decimal import Decimal
 import tempfile
 import gzip
+import numpy
 import xml.etree.ElementTree as ElementTree
 import zipfile
 import shutil
 from subprocess import check_call
+from returnn import SprintCache
```

Comment:
You cannot do the import like this.
You would first need to add sth like this:
```
my_dir = os.path.dirname(os.path.abspath(__file__))
returnn_dir = os.path.dirname(my_dir)
sys.path.insert(0, returnn_dir)
```
And then you can do:
```
import SprintCache
```
See the other tools.

Reply:
I added the import as suggested.